addresses/11155111.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ZORA_NFT_CREATOR_V1": "0x96315ee3C58d16A40DA8Ee05008bA1F6654ea358",
+  "ZORA_NFT_CREATOR_PROXY": "0x87cfd516c5ea86e50b950678CA970a8a28de27ac",
+  "ZORA_ERC721_TRANSFER_HELPER": "0x0000000000000000000000000000000000000000",
+  "EDITION_METADATA_RENDERER": "0xC5c958a65656A84b74100D1d420a1819fEA18d41",
+  "DROP_METADATA_RENDERER": "0x0Cf8733DEd6d9E0905A8cCc8DC767F381A76970a",
+  "ERC721_DROP": "0xF706f5A8030C1e7D344D2Ef32Dff6B8568B3d07B",
+  "FACTORY_UPGRADE_GATE": "0x3C1ebcF36Ca9DD9371c9aA99c274e4988906c6E3",
+  "OWNED_SUBSCRIPTION_MANAGER": "0x9B414A3F7872bdd2E6513689214BD2Debbe48340",
+  "MINT_FEE_RECIPIENT": "0xE51051a204afb2fC071A2406914cdEA5638e5018",
+  "MINT_FEE_AMOUNT": "111000000000000"
```

Comment:
makes sense to me to keep them in the same place.

Reply:
yeah it would be nice to separate value generated by the script vs configuration values, or have some sort of documentation/explanation to where these values come from.